### DISCUSSION TITLE
How to perform streaming queries using http protocol through DORIS REST API?

### DISCUSSION THREAD

**Discussion by gREpLT** [apache/doris#33384] [A - General / Q&A]
I want to query the data inside doris through http protocol, but because the data is too large, I have to use streaming query

Now I am using the 8030 port of DORIS for query, which obviously cannot achieve the results I expected. 

By the way, how about using PyMysql to directly connect to Doris' 9030 port?

**Comment by yagagagaga** [comment]
Maybe you can try spark-doris-connector for massive data import and export.
